mithril-infra/mithril.monitoring.tf

```diff
@@ -23,6 +23,9 @@ resource "null_resource" "mithril_monitoring" {
 
   provisioner "remote-exec" {
     inline = [
+      "mkdir -p /home/curry/data/monitoring",
+      "mkdir -p /home/curry/data/monitoring/prometheus",
```

Comment:
These 3 lines could be replaced by a single 
`mkdir -p /home/curry/data/monitoring/{prometheus,loki}`

Reply:
Done :heavy_check_mark: